recipes/butteraugli/all/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.4)
```

CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS requires CMake 3.4 
https://cmake.org/cmake/help/v3.4/variable/CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS.html